recipe/meta.yaml

```diff
@@ -12,9 +12,9 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
-  skip: true  # [py>=32]
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  number: 1
+  script: "{{ PYTHON }} --version"  # [py>=32]
```

Comment:
I’m not totally following what the purpose of this line is.

Reply:
The purpose of this line is to create a dummy package

Reply:
Can I have an empty script section and no build.sh file?

Reply:
I didn't know I could have no script section. So I removed the `--version` line.